core/ledger/kvledger/pkg_test.go

```diff
@@ -41,15 +46,57 @@ func newTestEnv(t testing.TB) *testEnv {
 	return createTestEnv(t, path)
 }
 
-func createTestEnv(t testing.TB, path string) *testEnv {
+func couchDBSetup() (addr string, cleanup func()) {
```

Comment:
why would the abstract kvledger package have couch configuration?

Reply:
I remove it